docs/yaml/functions/_build_target_base.yaml

```
@@ -292,16 +292,17 @@ kwargs:
       If the target is an [[executable]] this defaults to "bin", the only
       allowed value.
 
-      If it is a [[static_library]] it defaults to "lib", and may be "lib",
-      "staticlib", or "rlib". If "lib" then Rustc will pick a default, "staticlib"
-      means a C ABI library, "rlib" means a Rust ABI.
+      If it is a [[static_library]] it defaults to "lib", and may be "lib", "clib"
+      "staticlib", or "rlib". "clib" and "staticlib" means a C ABI library,
+      "lib" and "rlib" means a Rust ABI.
 
-      If it is a [[shared_library]] it defaults to "lib", and may be "lib",
-      "dylib", "cdylib", or "proc-macro". If "lib" then Rustc will pick a
-      default, "cdylib" means a C ABI library, "dylib" means a Rust ABI, and
-      "proc-macro" is a special rust procedural macro crate.
+      If it is a [[shared_library]] it defaults to "lib", and may be "lib", "clib",
+      "dylib", "cdylib", or "proc-macro". "clib" and "cdylib" means a C ABI library,
+      "lib" and "dylib" means a Rust ABI, and "proc-macro" is a special rust
```

```suggestion
      "lib" and "dylib" means a Rust ABI, and "proc-macro" is a special Rust
```

Should probably `s/rust/Rust/` everywhere at some point